src/App.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Route } from 'react-router-dom';
+import HomePage from './components/HomePage.jsx';
```

Comment:
The `.jsx` extension is not necessary on the imports.
Kindly remove it for all other imports.

Reply:
Noted 👍